test/Microsoft.TestPlatform.TestHostProvider.UnitTests/Hosting/DotnetTestHostManagerTests.cs

```diff
@@ -519,7 +519,7 @@ public void GetTestHostProcessStartInfoShouldThrowExceptionWhenDotnetIsNotInstal
 
             char separator = ';';
             var dotnetExeName = "dotnet.exe";
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (!System.Environment.OSVersion.Platform.ToString().StartsWith("Win"))
```

Comment:
Had to change this, otherwise I was getting conflict because `RuntimeInformation` was coming from multiple different assemblies on Linux build.